core/commands/partials/terraform_configurator.rb

```diff
@@ -184,18 +185,28 @@ def configure_nodes(nodes)
     Result.ok('')
   end
 
+  def retrieve_all_network_settings(nodes)
+    @all_nodes_network_settings = nodes.map do |node|
+      @ui.info("Retrieve network settings for node '#{node}'")
+      TerraformService.resource_network(node, @ui, @config.path).and_then do |node_network|
+        result = {
+            'keyfile' => node_network['key_file'],
+            'private_ip' => node_network['private_ip'],
+            'network' => node_network['public_ip'],
+            'whoami' => node_network['user'],
+            'hostname' => node_network['hostname']
+        }
+        Result.ok(result)
```

Comment:
The Result.ok() is not relevant here. You will always be able to go to the next step.